modules/core/src/core/lib/layer.js

```diff
@@ -362,6 +362,71 @@ export default class Layer {
     }
   }
 
+  // This method overwrites the current buffer and is used for multi-picking
```

Comment:
Better comment: `// Sets the specified instanced picking color to null picking color. Used for multi picking"`?